### ISSUE TITLE
owncloud, owncloudcmd wrapper on rh6: please fix the argument passing with $*

### ISSUE THREAD

**Issue by moscicki** [owncloud/client#3212]
It is impossible to pass option due to missing space between owncloud and $* :

``` 
> cat `which owncloud`

...
exec /opt/qt-4.8/bin/owncloud$*

# end
```

The result is:
```
>cernbox --help
/usr/bin/cernbox: line 8: /opt/qt-4.8/bin/cernbox--help: No such file or directory
/usr/bin/cernbox: line 8: exec: /opt/qt-4.8/bin/cernbox--help: cannot execute: No such file or directory

```

**Comment by jnweiger** [comment]
Gah, that's been there for ages. Thanks for spotting. 
Fix will sneak in for 1.8.1 already, as the packages have not yet been published by OBS.

**Comment by moscicki** [comment]
Thx!
